tests/test_models.py

```diff
@@ -100,3 +100,60 @@ def test_prophet_for_global_modeling():
     else:
         with pytest.raises(RuntimeError):
             results_train, results_test = benchmark.run()
+
+
+def test_seasonal_naive_model():
```

Comment:
could we split this into multiple test functions, or make this a parametrized test?
pytest has some cool features for that

Reply:
@ourownstory Amazing idea!